app/controllers/SubmissionController.scala

```diff
@@ -107,13 +108,13 @@ class SubmissionController @Inject() (
     response: HttpResponse,
     departureId: Option[String],
     auditType: AuditType
-  )(implicit hc: HeaderCarrier): Future[Result] = {
+  )(implicit hc: HeaderCarrier, request: VersionedRequest[JsValue]): Future[Result] = {
```

Comment:
Consider passing the phase in explicitly here rather than implicitly through the versioned request